### PR TITLE
feat: apply meta tags for seo per page

### PR DESCRIPTION
**Pull request by danielfpadilla**
#40 feat: apply meta tags for seo per page

Home Page
image

Team Page
image

Projects Page
image

### CODE REVIEW

File: src/constants/index.js
Comment (jofftiquez) [inline src/constants/index.js:1]: Please rename the file to `meta.js` or retain index but do not use `export default`. If you do that you can export the items separately like so:

```js
export default META_TITLE: '...';
export default META_DESCRIPTION: '....'
```

File: src/pages/index.vue [hunk 1]
```diff
@@ -70,14 +70,32 @@
 </template>
 
 <script>
-import { definePageMeta } from '#imports';
+import {
+  definePageMeta,
+  useSeoMeta,
+} from '#imports';
 import GenericPanel from '@/components/commons/GenericPanel.vue';
+import Constants from '~/constants';
```
Comment (jofftiquez) [inline src/pages/index.vue:78]: Use destructing. Also don't use Pascal case to JS files. It should only apply to component imports. Sorry this is not clear, but I will make a guide about this. Thanks

File: src/pages/index.vue [hunk 2]
```diff
@@ -70,14 +70,32 @@
...
 
 export default {
   components: {
     GenericPanel,
   },
-  setup() {
+
+  setup () {
+    const title = `Home - ${Constants.TITLE_DEFAULT}`;
```
Comment (jofftiquez) [inline src/pages/index.vue:86]: Update accordingly

File: src/pages/projects.vue [hunk 3]
```diff
@@ -6,10 +6,27 @@
 </template>
 
 <script>
-import { definePageMeta } from '#imports';
+import {
+  definePageMeta,
+  useSeoMeta,
+} from '#imports';
+import Constants from '~/constants';
```
Comment (jofftiquez) [inline src/pages/projects.vue:13]: Apply same fixes as above

File: src/constants/index.js
Comment (danielfpadilla) [reply src/constants/index.js:1]: I go with `meta.js` . It's way better. =)

File: src/pages/index.vue (hunk 1 above)
Comment (danielfpadilla) [reply src/pages/index.vue]: Applied changes using meta.js

File: src/pages/index.vue (hunk 2 above)
Comment (danielfpadilla) [reply src/pages/index.vue]: Applied changes.

File: src/pages/projects.vue (hunk 3 above)
Comment (danielfpadilla) [reply src/pages/projects.vue]: Destructing applied.

**Pull request merged**